lib/rules/declaration-block-no-redundant-longhand-properties/__tests__/index.js

```diff
@@ -79,6 +79,10 @@ testRule(rule, {
       code:
         "a { border-top-width: 1px; border-bottom-width: 1px; border-left-width: 1px; border-right-width: 1px; }",
       message: messages.expected("border-width")
+    },
+    {
```

Comment:
Can we please pull this out into its own `testRule` block with `syntax: styled`, please?

We should place it at the end of the file.